From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: disable some obex tests
Date: Tue, 30 Jan 2018 17:01:45 +0100

disable some obex tests as transient failures
reported by bug
https://bugzilla.suse.com/show_bug.cgi?id=1078285

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: bluez-5.71/Makefile.am
===================================================================
--- bluez-5.71.orig/Makefile.am
+++ bluez-5.71/Makefile.am
@@ -602,8 +602,8 @@ unit_test_gdbus_client_LDADD = gdbus/lib
 				src/libshared-glib.la $(GLIB_LIBS) $(DBUS_LIBS)
 
 if OBEX
-unit_tests += unit/test-gobex-header unit/test-gobex-packet unit/test-gobex \
-			unit/test-gobex-transfer unit/test-gobex-apparam
+unit_tests += unit/test-gobex-header unit/test-gobex-packet \
+			unit/test-gobex-apparam
 
 unit_test_gobex_SOURCES = $(gobex_sources) unit/util.c unit/util.h \
 						unit/test-gobex.c
